bucket/clamav.json

```diff
@@ -0,0 +1,50 @@
+{
+    "version": "0.102.4",
+    "description": "ClamAV is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats.",

```

Comment:
- Do not mention application name when it is same as manifest
- `Open source` is clear from license